pubspec.yaml

```diff
@@ -28,9 +29,14 @@ dev_dependencies:
   dart_dev: ^2.0.1
   dart_style: ^1.2.0
   dependency_validator: ^1.2.3
+  meta: ^1.1.7
   mockito: ^4.0.0
   pedantic: ^1.4.0
+  react: ^4.6.1
   test: ^1.5.1
 
 executables:
-  dart2_upgrade:
\ No newline at end of file
+  dart2_upgrade:
+  react_16_upgrade:
+  react_16_dependency_override_updater:
+  react_16_pubspec_upgrade:
```

Comment:
#nit should we remove the first underscore in these file names as well since we updated `react16_upgrade` in the other PR?